app/views/items/new.html.erb

```diff
@@ -3,58 +3,5 @@
       <h3>List an Item</h3>
     </div>
   </div>
-  <%= form_with model: @item, local: true do |f| %>
-    <div class="row">
-      <div class="col-md-3">
-        <%= f.label :name %>
-      </div>
-      <div class="col-md-4">
-        <%= f.text_field :name %>
-      </div>
-    </div>
-    <div class="row">
-      <div class="col-md-3">
-        <%= f.label :description %>
-      </div>
-      <div class="col-md-6">
-        <%= f.text_area :description %>
-      </div>
-    </div>
-    <div class="row">
-      <div class="col-md-3">
-        <%= f.label :price, in: 0.0..100000.0, step: 0.01 %>
-      </div>
-      <div class="col-md-2">
-        <%= f.number_field :price, in: 0.01..10000.00, step: 0.01 %>
-      </div>
-    </div>
-    <div class="row">
-      <div class="col-md-3">
-        <%= f.label :image %>
-      </div>
-      <div class="col-md-2">
-        <%= f.file_field :image %>
-      </div>
-    </div>
-    <div class="row">
-      <div class="col-md-3">
-        <%= f.label :deliverable %>
-      </div>
-      <div class="col-md-2">
-        <%= f.check_box :deliverable, {}, true, nil %>
-      </div>
-    </div>
-    <div class="row">
-      <div class="col-md-3">
-        <%= f.label :category %>
-      </div>
-      <div class="col-md-2">
-        <%= f.select :category, ['', 'Electronics', 'Books', 'Furniture']%>
-      </div>
-    </div>
-    <div class="row">
-      <div class="col-md-2">
-        <%= f.submit "Create new item" %>
-      </div>
-    </div>
-  <% end %>
+  
+  <%= render partial: 'layouts/item_form', locals: {button_text: "Save Changes"} %>
```

Comment:
good idea to replace this form w a partial, much cleaner this way :)

Reply:
thanks!